README.md

```diff
@@ -1,31 +1,130 @@
 # circulation-docker
- 
-This is the Docker image for Library Simplified's [Circulation Manager](https://github.com/NYPL-Simplified/circulation_manager).
+These are the Docker images for Library Simplified's [Circulation Manager](https://github.com/NYPL-Simplified/circulation_manager).
 
-## Building
+## Supported tags and respective `Dockerfile` links
 
-If you plan to work with stable versions of the Circulation Manager, we strongly recommend using the latest stable versions of circ-deploy and circ-scripts [published to Docker Hub](https://hub.docker.com/r/nypl/). However, there may come a time in development when you want to build Docker containers for a particular version of the Circulation Manager. If so, please use the instructions below.
```

Comment:
Maybe for consistency  call these `circ-deploy:` & `circ-scripts:`?

Reply:
I'm going to chat with some other Library Simplified folks about your suggestion to change the app container to `circ-webapp` instead of `circ-deploy`. I think it's a great idea. Thanks!

Reply:
For sure! Let me know how it goes.